### ISSUE TITLE
Refactor: Extract method results in syntax error in multiline "for"-statement

### ISSUE THREAD

**Issue by ThisIsAlsoMyPassword** [microsoft/pylance-release#1052]
## Environment data

-   VS Code version: 1.53.2
-   Extension version (available under the Extensions sidebar): v2021.2.633441544
-   OS and version: Windows 10, Version	10.0.19041 Build 19041
-   Python version: 3.8.5, 64bit
-   Value of the `python.languageServer` setting:  Pylance

## Expected behaviour
This code:
```Python
some_very_long_variable_forcing_a_line_break = []

for variable in [
    some_very_long_variable_forcing_a_line_break
    for some_very_long_variable_forcing_a_line_break in some_very_long_variable_forcing_a_line_break
    if some_very_long_variable_forcing_a_line_break == 1
]:
    pass
```
is supposed to be refactored into:
```Python
def new_func():
    some_very_long_variable_forcing_a_line_break = []
    for variable in [
    some_very_long_variable_forcing_a_line_break
    for some_very_long_variable_forcing_a_line_break in some_very_long_variable_forcing_a_line_break
    if some_very_long_variable_forcing_a_line_break == 1
]:
        pass

new_func()
```

## Actual behaviour
The actual new code contains an uneccessary new line before the colon and additional indentation, resulting in a syntax error:
```Python
def new_func():
    some_very_long_variable_forcing_a_line_break = []
    for variable in [
    some_very_long_variable_forcing_a_line_break
    for some_very_long_variable_forcing_a_line_break in some_very_long_variable_forcing_a_line_break
    if some_very_long_variable_forcing_a_line_break == 1
]
    :
        pass

new_func()
```

## Steps to reproduce:

1. copy minimal example above in new window
2. select all
3. click lightbulb icon
4. select "Extract method"

## Logs

No log output in "Python"

**Comment by judej** [comment]
Thanks for the report. Will investigate this.

**Comment by savannahostrowski** [comment]
Closing this issue as this should be fixed in 2020.5.3. Please ping me if it's not and I can reopen the issue!
